known-shortcuts/ticktick.ts

```diff
@@ -1,3 +1,5 @@
+import { AppParams } from "~/types";
+
 const app = {
```

Comment:
:zap: **Suggestion** 
The `app` object is being cast to `AppParams` using `as AppParams`. While this is a valid TypeScript pattern, it is generally safer to ensure that the object conforms to the `AppParams` type without casting. Consider defining the `app` object with the `AppParams` type directly to leverage TypeScript's type checking.